modules/installing-gitops-operator-using-cli.adoc

```diff
@@ -31,29 +92,52 @@ spec:
 <2> Specify the name of the Operator to subscribe to.
 <3> Specify the name of the CatalogSource that provides the Operator.
 <4> The namespace of the CatalogSource. Use `openshift-marketplace` for the default OperatorHub CatalogSources.
-+
+
 . Apply the `Subscription` to the cluster:
 +
 [source,terminal]
 ----
 $ oc apply -f openshift-gitops-sub.yaml
 ----
-. After the installation is complete, ensure that all the pods in the `openshift-gitops` namespace are running:
++
+.Example output
+[source,terminal]
+----
+subscription.operators.coreos.com/openshift-gitops-operator created
+----
+
+. After the installation is complete, verify that all the pods in the `openshift-gitops` namespace are running:
 +
 [source,terminal]
 ----
 $ oc get pods -n openshift-gitops
 ----
++
 .Example output
+[source,terminal]
+----
+NAME                                                      	  READY   STATUS    RESTARTS   AGE
+cluster-b5798d6f9-zr576                                   	  1/1 	  Running   0          65m
+kam-69866d7c48-8nsjv                                      	  1/1 	  Running   0          65m
+openshift-gitops-application-controller-0                 	  1/1 	  Running   0          53m
+openshift-gitops-applicationset-controller-6447b8dfdd-5ckgh       1/1 	  Running   0          65m
+openshift-gitops-dex-server-569b498bd9-vf6mr                      1/1     Running   0          65m
```

Comment:
Hello @jaideepr97, I added `openshift-gitops-dex-server-` as an example pod after the user uses the `oc get pods -n openshift-gitops` command, as I could see that pod when testing this. Is it okay to include?

Reply:
that's fine.